Tests/NIOHTTP2Tests/SimpleClientServerTests.swift

```diff
@@ -1230,4 +1230,90 @@ class SimpleClientServerTests: XCTestCase {
         XCTAssertNoThrow(try self.clientChannel.finish())
         XCTAssertNoThrow(try self.serverChannel.finish())
     }
+
+    func testManyConcurrentInactiveStreams() throws {
+        let maxCachedClosedStreams = 1024
+
+        // Begin by getting the connection up.
+        try self.basicHTTP2Connection(maxCachedClosedStreams: maxCachedClosedStreams)
```

Comment:
I was about to mention the following (first part is already fixed): As you are declaring `maxCachedClosedStreams` above, should you replace `1024` with `maxCachedClosedStreams` here? Also, would it make sense to pick a lower value than 1024 for faster test runtime?